### PR TITLE
[PlayerList] Support for UserInfo server message.

### LINKED ISSUE #2010: Playerlist player multiplication
Screenshots are failing, but upon connecting to MM game, KingStad who was spec also appeared as player on team 1

### PR DESCRIPTION
**Pull request by DeinFreund**

### CODE REVIEW

File: LuaUI/cawidgets.lua [hunk 1]
```diff
@@ -224,6 +225,7 @@ local callInLists = {
   'TextCommand',
   'CommandNotify',
   'AddConsoleLine',
+  'ReceiveUserInfo',
```
Comment (sprunk) [inline LuaUI/cawidgets.lua]: I'd be wary of putting it like this because it's not an engine call. At least document this.

Comment (DeinFreund) [inline LuaUI/cawidgets.lua]: Document where? If we have a documentation about this it'd be nice to know..

Comment (sprunk) [inline LuaUI/cawidgets.lua]: inline

Comment (DeinFreund) [inline LuaUI/cawidgets.lua]: done

**Pull request merged**